crates/bcr-wallet-core/src/pocket.rs

```diff
@@ -73,11 +72,11 @@ impl<Repo> CrPocket<Repo>
 where
     Repo: PocketRepository,
 {
-    fn validate_keysets<'a>(
+    fn validate_keysets<'inf>(
```

Comment:
haha, I'm so used to `'a` and `'b` that this looks foreign to me, even though it provides more context 😅